### PR TITLE
[13.0] [MIG] base_import_module_group

### PR DESCRIPTION
**Pull request by yankinmax**

### CODE REVIEW

File: base_import_module_group/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# Copyright 2020 Camptocamp SA
+# Copyright 2021 Camptocamp SA
```
Comment (grindtildeath) [inline base_import_module_group/__manifest__.py]: This shouldn't be changed as the module was created in 2020.
You can still change to `Copyright 2020-2021 Camptocamp SA` if you wish

File: base_import_module_group/models/ir_module.py [hunk 2]
```diff
@@ -1,14 +1,13 @@
-# Copyright 2020 Camptocamp SA
+# Copyright 2021 Camptocamp SA
```
Comment (grindtildeath) [inline base_import_module_group/models/ir_module.py]: Same as above

**Review by grindtildeath — COMMENTED** [review]
LG, would be great to add a small test to improve coverage though ;)

**Comment by grindtildeath** [comment]
Can you please cherry-pick your commit from https://github.com/OCA/server-tools/pull/2123 instead of having the test added in the migration commit? :pray:

**Comment by yankinmax** [comment]
sure)

File: base_import_module_group/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "name": "Import Module Group",
+    "summary": "Restrict module importation to specific group",
+    "version": "13.0.1.0.0",
+    "category": "Uncategorized",
+    "website": "https://github.com/OCA/server-tools",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "auto-install": True,
```
Comment (thomaspaulb) [inline base_import_module_group/__manifest__.py]: There is a typo here, see [this PR](https://github.com/OCA/server-tools/pull/2367)

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**